src/partials/collection.html

```diff
@@ -1,3 +1,53 @@
 <section class="collection">
-
+    <h2 class="collection-title">Our collection</h2>
+    <ul class="collection-list">
+        <li class="collection-item">
+            <img srcset="../img/collection/poltava@2x.jpg 2x" src="../img/collection/poltava.jpg"
```

Comment:
Change path with 1 dot in for all paths below.
exmpl: "../img/collection/poltava@2x.jpg 2x"  => "./img/collection/poltava@2x.jpg 2x" for general standard

Reply:
Done